lib/ansible/modules/windows/win_chocolatey_source.ps1

```diff
@@ -25,6 +25,7 @@ $priority = Get-AnsibleParam -obj $params -name "priority" -type "int"
 $source = Get-AnsibleParam -obj $params -name "source" -type "str" -failifempty ($state -ne "absent")
 $source_username = Get-AnsibleParam -obj $params -name "source_username" -type "str"
 $source_password = Get-AnsibleParam -obj $params -name "source_password" -type "str" -failifempty ($null -ne $source_username)
+$source_api = Get-AnsibleParam -obj $params -name "source_api" -type "str"
```

Comment:
Probably just best to have this as `api_key` unless there are multiple types of api keys that it could refer to?